lib/robots/sdr_repo/preservation_ingest/update_catalog.rb

```diff
@@ -1,6 +1,6 @@
 require 'faraday'
 require 'retries'
-
+require 'byebug'
```

Comment:
?

Reply:
assumed this is useful for debugging purposes during development, but yes, not sure why its in the class now that you mention it

Reply:
fixed.